opsdroid/parsers/mention.py

```diff
@@ -0,0 +1,37 @@
+"""A helper function for parsing and executing mention skills."""
+
+import logging
+
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def parse_mention(opsdroid, skills, message):
+    """Parse a message if the bot is mentioned."""
```

Comment:
Could we turn this into  a google style docstring?

Reply:
Could you give an example? Unfortunately I don't know the definition of a google style docstring

Reply:
The dialogflow parser has an example of google style docstring on [this line] (https://github.com/opsdroid/opsdroid/blob/7282a5da065cbe73212245f55b7cc4b3537ebd37/opsdroid/parsers/dialogflow.py#L13)

I've added some explanation but we just need to add this:
```
    Args:
        opsdroid (OpsDroid): An instance of opsdroid.core.
        skills (list): A list containing all skills available.
        message(object): An instance of events.message.
        config (dict): configuration settings from the
            file config.yaml.
    Return:
        Either empty list or a list containing all matched skills.
```